Trustroots/trustroots
# Experiences: redesign success messages

#### Proposed Changes

* Redesign "done" and "you've already left reference" messages
* Shaping copy towards "experiences"

**TODO:** 
Tests still seem to need adjusting but this is good for functional/code/copy review.


### Screenshots
**"Done" before**

<img width="712" alt="Screenshot 2020-12-06 at 23 28 14" src="https://user-images.githubusercontent.com/87168/101293042-dd1efc80-381b-11eb-8788-3bfc53b3c09b.png">


**"Done" after**

<img width="682" alt="Screenshot 2020-12-06 at 23 19 18" src="https://user-images.githubusercontent.com/87168/101293025-d09aa400-381b-11eb-9a3c-0bd3e64d63a6.png">

**"Already" before**

<img width="710" alt="Screenshot 2020-12-06 at 23 27 48" src="https://user-images.githubusercontent.com/87168/101293045-e14b1a00-381b-11eb-848f-42b08dfdf961.png">

**"Already" after**

<img width="703" alt="Screenshot 2020-12-06 at 23 26 11" src="https://user-images.githubusercontent.com/87168/101293034-d6908500-381b-11eb-9f83-5ce9da5dc5ba.png">


#### Testing Instructions

* Leave reference to someone, with and without reporting them
* Try the same again and see "already wrote" message
